docs/development/index.rst

```diff
@@ -0,0 +1,41 @@
+:orphan:
+
+***********************
+Contributing to Astropy
+***********************
+
+So you are interested in contributing to the Astropy Project?  Excellent! We
+love contributions! Astropy is open source, built on open source, and we'd
+love to have in our community.
+
+For the time being, please see our `CONTRIBUTING.md
```

Comment:
As above, my take is to have this section focused on writing code and docs, so skip the generic "contributing" thing, which is well-covered in other places.